packages/base/src/config/Icons.ts

```diff
@@ -3,16 +3,17 @@ import { getIconCollectionByAlias } from "../assets-meta/IconCollectionsAlias.js
 
 const IconCollectionConfiguration = new Map<string, string>();
 
-// All supported icon collections + uknown custom ones
-type IconCollection = "SAP-icons" | "SAP-icons-v4" | "SAP-icons-v5" | "horizon" | "tnt" | "tnt-v2" | "tnt-v3" | "business-suite" | string;
+// All supported icon collections + unknown custom ones
```

Comment:
"business-suite" is still part of the supported collections - this is the "Auto" mode and must be part of the type IconCollection - "business-suite-v1" | "business-suite-v2" come in addition